docs/nodes/nodes-library/nodes/Spotify/README.md

```diff
@@ -0,0 +1,63 @@
+---
+permalink: /nodes/n8n-nodes-base.spotify
+---
+
+# Spotify
+
+[Spotify](https://www.spotify.com/) is a music streaming service containing millions of music tracks and podcasts. Spotify allows users to create and manage their own playlists, explore new music through recommendation services, and listen to songs on demand.
+
+::: tip 🔑 Credentials
+You can find authentication information for this node [here](../../../credentials/Spotify/README.md).
+:::
+
+## Basic Operations
+
+- Album
+    - Get an album
+    - Get an album's tracks
+- Artist
+    - Get an artist
+    - Get an artist's albums
+    - Get an artist's related artists
+    - Get an artists's top tracks
+- Player
+    - Add song to queue
+    - Get the currently playing song
+    - Go to the next song
+    - Pause your music
+    - Go to the previous song
+    - Get the recently played song
+    - Start your music
+- Playlist
+    - Add an item to a playlist
+    - Get a playlist
+    - Get a playlist's tracks
+    - Get a user's playlists
+    - Remove an item from a playlist
+- Track
+    - Get a track
+    - Get a track's audio features
```

Comment:
Could you make sure that this matches the descriptions in the node?
![image](https://user-images.githubusercontent.com/48641776/86734585-b92e7380-c04f-11ea-85d8-bb939a0987f0.png)